tests/flytekit/unit/annotated/test_type_hints.py

```diff
@@ -273,13 +273,13 @@ def t1() -> datetime.datetime:
         return datetime.datetime.now()
 
     @workflow
-    def my_wf() -> str:
+    def my_wf() -> FlyteSchema:
         dt = t1()
         return sql(ds=dt)
 
     with task_mock(sql) as mock:
-        mock.return_value = "Hello"
-        assert my_wf() == "Hello"
+        mock.return_value = pandas.DataFrame(data={"x": [1, 2], "y": ["3", "4"]})
+        assert (my_wf().open().all() == pandas.DataFrame(data={"x": [1, 2], "y": ["3", "4"]})).all().all()
```

Comment:
is this the right way to use the FlyteSchema?

Reply:
yes, but you cannot just compare 2 pandas dataframes like this
you have to compare, `==` which returns a new dataframe, that you assert has all `True`

Look at some examples of how I am doing it